frontend/src/js/store/appSlice/selectors.ts

```diff
@@ -18,10 +18,9 @@ const getAppDocsVersion = state => state.app.docsVersion;
 export const getFeatures = state => state.app.features;
 export const getFullVersionInformation = state => state.app.versionInformation;
 
-export const getDocsVersion = createSelector([getAppDocsVersion, getFeatures], (appDocsVersion, { isHosted }) => {
+export const getDocsVersion = createSelector([getAppDocsVersion], appDocsVersion => {
   // if hosted, use latest docs version
-  const docsVersion = appDocsVersion ? `${appDocsVersion}/` : 'development/';
-  return isHosted ? '' : docsVersion;
+  return appDocsVersion ? `${appDocsVersion}/` : '';
```

Comment:
Once you are here, maybe remove the whole `appDocsVersion`? From Mender Server 4.0 on, even on o-prem we should always server docs.mender.io (iow there is no docs.mender.io/4.0)